packages/chain-adapters/src/ChainAdapterManager.ts

```diff
@@ -2,7 +2,7 @@ import { ChainTypes } from '@shapeshiftoss/types'
 import * as unchained from '@shapeshiftoss/unchained-client'
 
 import { ChainAdapter } from './api'
-import * as bitcoin from './bitcoin'
+import * as bitcoin from './utxo'
```

Comment:
probably some imports that will have to be fixed for this, but we should keep the naming consistent

```suggestion
import * as utxo from './utxo'
```